### PR TITLE
Fix message wrapping on PR commit list

### PR DESCRIPTION
**Pull request by CirnoT**
Before:
chrome_2020-05-25_03-31-35

After:
chrome_2020-05-25_03-30-56

The example has exactly 100 characters but it's perfectly possible to remote-branch merges for branches with long names to take more, personally I have noticed it for message with 120 characters.

>Merge branch 'very-very-long-branch_name' of git.some.domain.com:company/repository into very-very-long-branch_name

### CODE REVIEW

**Comment by zeripath** [comment]
I wonder if we should just have these truncate rather than wrap?

**Comment by CirnoT** [comment]
They are truncated when multi-line.

**Review by zeripath — APPROVED** [review]

File: templates/repo/commits_list_small.tmpl [hunk 1]
```diff
@@ -33,18 +33,16 @@
...
 				</span>
 			{{end}}
 		</span>
 
-		<span class="message-wrapper">
-			{{ $commitLink:= printf "%s/%s/%s/commit/%s" AppSubUrl  $.Issue.PullRequest.BaseRepo.OwnerName $.Issue.PullRequest.BaseRepo.Name .ID }}
-			<span class="mono commit-summary has-emoji{{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
-		</span>
+		{{ $commitLink:= printf "%s/%s/%s/commit/%s" AppSubUrl  $.Issue.PullRequest.BaseRepo.OwnerName $.Issue.PullRequest.BaseRepo.Name .ID }}
+		<span class="mono commit-summary has-emoji{{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
```
Comment (silverwind) [inline templates/repo/commits_list_small.tmpl]:
```suggestion
		<span class="mono commit-summary {{if gt .ParentCount 1}} grey text{{end}}" title="{{.Summary}}">{{RenderCommitMessageLinkSubject .Message ($.Issue.PullRequest.BaseRepo.Link|Escape) $commitLink $.Issue.PullRequest.BaseRepo.ComposeMetas}}</span>
```

Might as well remove this unused class here (was previously used by JS emoji conversion). This looks to be the last occurence in the codebase.

Comment (mrsdizzie) [reply templates/repo/commits_list_small.tmpl]: suck back in from code copied before that PR and committed after

**Review by silverwind — APPROVED** [review]

**Review by lafriks — APPROVED** [review]

**Comment by techknowlogick** [comment]
please resolve conflicts

**Comment by zeripath** [comment]
make lg-tm work

**Pull request merged**